doc/reference/tooling/tt_cli/connect.rst

```diff
@@ -28,6 +28,20 @@ Options
 
     ``-`` – read the script from stdin.
 
+.. option:: -i, --interactive
```

Comment:
There are a bit more `tt connect` flags:

```
      --sslcafile string      path to a trusted certificate authorities (CA) file
      --sslcertfile string    path to an SSL certificate file
      --sslciphers string     colon-separated (:) list of SSL cipher suites the connection
      --sslkeyfile string     path to a private SSL key file
```

Or does it not related to the issue?

Reply:
Thanks, added these options and missing ``tt connect`` auth methods (URI string and env variables).